prime-dotnet-webapi/ViewModels/Enrollee/EnrolleeListViewModel.cs

```diff
@@ -57,5 +58,16 @@ public string CurrentTOAStatus
         public string AdjudicatorIdir { get; set; }
 
         public bool AlwaysManual { get; set; }
+
+        [JsonIgnore]
+        public ICollection<EnrolleeRemoteUser> EnrolleeRemoteUsers { get; set; }
+
+        public bool RemoteAccess
+        {
+            get
+            {
+                return (EnrolleeRemoteUsers.Count > 0) ? true : false;
```

Comment:
You don't need to do a ternary here, you can just do
`return (EnrolleeRemoteUsers.Count > 0);`